sdk/python/tests/compiler/component_builder_test.py

```diff
@@ -247,22 +263,25 @@ def sample_component_func_two(a: str, b: int) -> float:
     result = float(b + 5)
   return result
 
-def wrapper_sample_component_func_two(a,b,_output_file):
-  output = sample_component_func_two(str(a),int(b))
-  import os
-  os.makedirs(os.path.dirname(_output_file))
-  with open(_output_file, "w") as data:
-    data.write(str(output))
-
 import argparse
-parser = argparse.ArgumentParser(description="Parsing arguments")
-parser.add_argument("a", type=str)
-parser.add_argument("b", type=int)
-parser.add_argument("_output_file", type=str)
-args = vars(parser.parse_args())
-
-if __name__ == "__main__":
-  wrapper_sample_component_func_two(**args)
+_missing_arg = object()
+_parser = argparse.ArgumentParser(prog='Sample component func two', description='')
+_parser.add_argument("--a", dest="a", type=str, required=True, default=_missing_arg)
+_parser.add_argument("--b", dest="b", type=int, required=True, default=_missing_arg)
+_parser.add_argument("----output-paths", dest="_output_paths", type=str, nargs=1)
```

Comment:
should this be --output-path?

Reply:
This is not a typo. I used four dashes for the output-paths so that this parameter never clashes with the user's imput parameter names.